src/context/movie-details-context/MovieDetailsContextProvider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { createContext, FC, useContext, useState } from 'react'
+import { IMovieDetailsContext } from './movieDetailsContextTypes'
+import { IMovieItem } from '../../pages/Home/components/movie-card'
+
+const MovieDetailsContext = createContext({
+    movieInfo: {} as IMovieItem,
```

Comment:
Make `movieInfo` optional to avoid type casting.

Reply:
created interface for createContext